core/src/main/java/zipkin/reporter/AsyncReporter.java

```diff
@@ -147,7 +147,7 @@ public Builder queuedMaxBytes(int queuedMaxBytes) {
       if (messageTimeoutNanos > 0) { // Start a thread that flushes the queue in a loop.
         final BufferNextMessage consumer =
             new BufferNextMessage(sender, messageMaxBytes, messageTimeoutNanos);
-        new Thread(() -> {
+        final Thread flushThread = new Thread(() -> {
```

Comment:
i was going to say this project had to be Java6 compatible, but then discovered its use of retrolambda – how cool is that?!